filament/backend/src/vulkan/PlatformVkLinuxWayland.cpp

```diff
@@ -63,4 +67,11 @@ void* PlatformVkLinuxWayland::createVkSurfaceKHR(void* nativeWindow, void* insta
     return surface;
 }
 
+void PlatformVkLinuxWayland::getSwapChainFallbackExtent(void* nativeWindow, uint32_t* width,
+        uint32_t* height) noexcept {
+    wl* ptrval = reinterpret_cast<wl*>(nativeWindow);
+    *width = ptrval->width;
+    *height = ptrval->height;
```

Comment:
why do we need a new virtual and not do this during create swapchain?

Reply:
I think the issue is that `createVkSurfaceKHR` should return a type that we define (that can contain the width/height etc) + a pointer to the native surface -- instead of returning a pointer to the native surface directly as a void*.

Reply:
done.  

I think my initial thoughts are not to make `createVkSurfaceKHR` do more things than necessary.  But I guess I'm passing the extent in `nativeWindow` anyways.